numpy/core/tests/test_multiarray.py

```diff
@@ -1157,9 +1157,11 @@ def test_structuredscalar_indexing(self):
     def test_multiindex_titles(self):
         a = np.zeros(4, dtype=[(('a', 'b'), 'i'), ('c', 'i'), ('d', 'i')])
         assert_raises(KeyError, lambda : a[['a','c']])
-        assert_raises(KeyError, lambda : a[['b','b']])
+        assert_raises(KeyError, lambda : a[['a','a']])
+        assert_raises(ValueError, lambda : a[['b','b']])  # field exists, but repeated
```

Comment:
I think this is a new error as of 1.14, so we can change it in 1.14.1 without compatibility concerns